recipes/fastpca/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "FastPCA" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/moldyn/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 67aebbbfb637f8a0fdb1ae130e3f457b3a21fc982cb99de7e61b4fe03dcbd0ae
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - boost
```

Comment:
this should probably be a host dependency and maybe just boost-cpp

Reply:
I tried it in host first and the build fails.

Reply:
can try boost-cpp